src/components/ReportingConfig/ReportingConfigForm.jsx

```diff
@@ -308,12 +308,12 @@ class ReportingConfigForm extends React.Component {
           for="pgpEncyptionKey"
           helpText="The key for encyption, if PGP encrypted file is required"
         >
-          <label htmlFor="pgpEncryptionKey">PGP Ecryption Key</label>
+          <label htmlFor="pgpEcryptionKey">PGP Ecryption Key</label>
```

Comment:
Is it intentionally `Ecryption` vs. `Encryption`??

Reply:
Oops, good catch. I copy/pasted the incorrect one throughout :) Updating now!